applications/nrf_desktop/doc/usb_state_pm.rst

```diff
@@ -0,0 +1,34 @@
+.. _nrf_desktop_usb_state_pm:
+
+USB state power manager module
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |usb_state_pm| is small, stateless module, that manages power state required by
+the USB state module.
+
+Configuration
+*************
+
+The module is enabled by selecting :kconfig:`CONFIG_DESKTOP_USB_PM_ENABLE`.
+It depends on :kconfig:`CONFIG_DESKTOP_USB_ENABLE` and :kconfig:`CONFIG_CAF_POWER_MANAGER`.
+
+There is no other configuration.
```

Comment:
I am not sure these two lines are needed.

Reply:
I would leave the line that describes that logs are inherited from USB state module. If anybody would look for that setting. The information that there is no other configuration may be removed.